codegen/src/main/kotlin/apifi/parser/PathsParser.kt

```diff
@@ -31,4 +32,17 @@ object PathsParser {
     private fun operationSpecifier(operation: io.swagger.v3.oas.models.Operation, httpMethod: PathItem.HttpMethod, endpoint: String) =
         (operation.operationId
             ?: (httpMethod.toString() + endpoint.replace(Regex("[^A-Za-z ]"), " ")).toTitleCase())
-}
\ No newline at end of file
+
+    /**
+     * Must start with a lower case letter and use the camel case and no underscores.
+     */
+    private fun getValidOperationId(operationId: String?): String? {
+        return operationId?.let {
+            if (it.first().isDigit()) {
+                operationId.substring(1).toCamelCase()
```

Comment:
how about adding operation at the start if the value doesn't start with a letter?
`1 list All_Pets ` would become `operation1ListAllPets`

Reply:
hi @niraj8, sure thing, no problem. I'll implement that.

Reply:
hi @niraj8 the change is implemented.